validator-network/src/network_impl.rs

```diff
@@ -128,11 +128,11 @@ where
                 Ok(peer_id)
             }
             Ok(None) => {
-                log::trace!(validator_id, %public_key, "Unable to resolve validator peer ID: Entry not found in DHT");
+                log::debug!(validator_id, %public_key, "Unable to resolve validator peer ID: Entry not found in DHT");
```

Comment:
I think it'd be better to still log these messages the first time they appear, but stop logging them afterwards.